### PR TITLE
Account model

### PR DESCRIPTION
**Pull request by ZeRoGerc**

### CODE REVIEW

File: app/src/main/java/ly/loud/loudly/application/Loudly.java [hunk 1]
```diff
@@ -184,6 +192,19 @@ public KeyKeeper getKeyKeeper(int network) {
      * @param keyKeeper KeyKeeper, that should be stored
      */
     public void setKeyKeeper(int network, KeyKeeper keyKeeper) {
+        if (keyKeeper != null) {
+            getAppComponent().coreModel().connectToNetworkById(network, keyKeeper)
```
Comment (danilkolikov) [inline app/src/main/java/ly/loud/loudly/application/Loudly.java]: Overhead

File: app/src/main/java/ly/loud/loudly/application/models/FacebookModel.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    @Inject
+    public FacebookModel(@NonNull Loudly loudlyApplication) {
+        this.loudlyApplication = loudlyApplication;
+        loadFromDB();
+    }
+
+    /**
+     * Load wrap from DataBase
+     */
+    private void loadFromDB() {
```
Comment (danilkolikov) [inline app/src/main/java/ly/loud/loudly/application/models/FacebookModel.java]: Now we're loading all networks in one action

File: app/src/main/java/ly/loud/loudly/application/Loudly.java (hunk 1 above)
Comment (ZeRoGerc) [inline app/src/main/java/ly/loud/loudly/application/Loudly.java]: fixed

File: app/src/main/java/ly/loud/loudly/application/models/NetworkContract.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+//TODO: nonnull annoations
+public interface NetworkContract {
+
+    /**
+     * Upload image to network
+     * @param image - image to load
+     * @return id of image in given network
+     */
+    @CheckResult
+    Single<Long> upload(Image image);
```
Comment (danilkolikov) [inline app/src/main/java/ly/loud/loudly/application/models/NetworkContract.java]: Replace Long to String, networks return IDs as strings

**Comment by danilkolikov** [comment]
ok

**Pull request merged**